src/main/kotlin/net/atos/zac/admin/ZaakafhandelParameterBeheerService.kt

```diff
@@ -67,25 +66,23 @@ class ZaakafhandelParameterBeheerService @Inject constructor(
         return entityManager.createQuery(query).resultList
     }
 
-    fun createZaakafhandelParameters(zaakafhandelParameters: ZaakafhandelParameters): ZaakafhandelParameters {
+    fun storeZaakafhandelParameters(zaakafhandelParameters: ZaakafhandelParameters): ZaakafhandelParameters {
         zaakafhandelParameterService.clearListCache()
         ValidationUtil.valideerObject(zaakafhandelParameters)
         zaakafhandelParameters.apply {
             humanTaskParametersCollection.forEach { ValidationUtil.valideerObject(it) }
             userEventListenerParametersCollection.forEach { ValidationUtil.valideerObject(it) }
             mailtemplateKoppelingen.forEach { ValidationUtil.valideerObject(it) }
-            creatiedatum = ZonedDateTime.now()
+            creatiedatum = zaakafhandelParameters.creatiedatum ?: ZonedDateTime.now()
```

Comment:
Doesn't make sense setting the `creatiedatum` here, as it might be an update... I'd move this to the logic for creating the ZAPs.